libr/util/file.c

```diff
@@ -365,6 +365,11 @@ R_API char *r_file_slurp(const char *str, R_NULLABLE size_t *usz) {
 				}
 				size += r;
 			} while (!feof (fd));
+			char *nbuf = realloc (buf, size + 1);
+			if (nbuf) {
```

Comment:
I think you should invert the condition and return Null when the reallocation fails.